### ISSUE TITLE
tree-sitter-disassembly issue

### ISSUE THREAD

**Issue by r3k2** [nvim-treesitter/nvim-treesitter#6204]
### Describe the bug

https://github.com/ColinKennedy/tree-sitter-disassembly is listed on the list but will not install

### To Reproduce

Add https://github.com/ColinKennedy/tree-sitter-disassembly to my list of treesitter must be installed list and says is not supported even do I got it from this git repo list.

### Expected behavior

to install

### Output of `:checkhealth nvim-treesitter`

```markdown
nvim-treesitter: 

- ERROR No healthcheck found for "nvim-treesitter" plugin.
```

### Output of `nvim --version`

```text
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1702233742

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

### Additional context

this is from nvchad

**Comment by clason** [comment]
Install nvim-treesitter first. Ask NvChad how.

**Comment by r3k2** [comment]
@clason Dude, you just like to be disrespectful, right? Did you read? It IS INSTALLED is by default I use TSInstall and TSUpdate etc and installed other "languages" with no problem!

**Comment by clason** [comment]
> ERROR No healthcheck found for "nvim-treesitter" plugin.

implies that nvim-treesitter is not installed, or not loaded. We can't support "distributions". If you can't reproduce an issue with a minimal config that _only_ involves this plugin, it's not on topic here.

**Comment by siduck** [comment]
@clason sorry for the noise created by that nvchad user. I always warn them not to bother plugin devs. Just using the master fixes the issue as it was using `0.9.2` tag in nvchad

. @r3k2 https://github.com/NvChad/NvChad/commit/8fe6a6560eff96238f55701470494ad911eab955

**Comment by lucario387** [comment]
ok 🔨 
One note: `TSInstall <tab>` for the list of parsers that you can actually download with your version of nvim-treesitter
